src/Plugin/facets/query_type/LocalGovDirectoriesQueryType.php

```diff
@@ -63,11 +64,48 @@ public function execute() {
    * {@inheritdoc}
    */
   public function build() {
+    $results = $this->results;
     $query_operator = $this->facet->getQueryOperator();
+    $field_identifier = $this->facet->getFieldIdentifier();
 
-    if (!empty($this->results)) {
+    // Get passed in facets.
+    $temp_query = $this->query->getOriginalQuery();
+    $temp_query->preExecute();
```

Comment:
What's the reason for getting the original query and running pre execute here.

I noticed it because by doing so it's running execute() of this LocalGovDirectoriesQueryType again which was all getting a bit circular.

Reply:
With preexecute
<img width="400" alt="Screenshot 2023-05-24 at 6 27 12 pm" src="https://github.com/localgovdrupal/localgov_directories/assets/1467480/1a0cc2e7-b901-46fa-8b6a-9fb5551fb415">


Without preexecute
<img width="408" alt="Screenshot 2023-05-24 at 6 27 26 pm" src="https://github.com/localgovdrupal/localgov_directories/assets/1467480/7bd67ea8-9ffc-4d84-add7-4b259d47f490">